src/pages/services/aem-cf-console-admin/api/action-bar/index.md

```diff
@@ -145,3 +145,67 @@ const guestConnection = await register({
 | Field    | Type                                                                        | Required | Description                                                                  |
 |----------|-----------------------------------------------------------------------------| ------ |------------------------------------------------------------------------------|
 | id       | `string`                                                                    | ✔️  | "id" is a "data-id" attribute of the button (please use browser developer console to find actual value). |
+
+### Get List of selected Fragments
+
+You can access data about the currently selected Content Fragments using the `host.fragmentSelections.getSelections()` method in an extension.
+
+
+## Examples:
+### Buttons conditional rendering
+
+Content Fragment Admin invokes getButtons method each time a user changes content fragments selection. Therefore, you can dynamically return different sets of buttons based on the data that has changed on the page. 
+
+For example:
+
+```js
+import { register } from "@adobe/uix-guest";
+
+// ...
+
+const guestConnection = await register({
+    id: "my.company.extension-with-action-bar-button",
+    methods: {
+        actionBar: {
+            async getButtons() {
+                const selections = await guestConnection?.host?.fragmentSelections?.getSelections?.() || [];
+                const defaultButtons = [
+                    {
+                        id: "default_btn",
+                        label: "Default Button",
+                        onClick(selections) {}
+                    }
+                ]
+
+                if (selections.length > 1 ) {
+                    return [
+                        ...defaultButtons,
+                        {
+                            id: "two_and_more_selected",
+                            label: "Ext two and more selected",
+                        }
+                    ]
+                }
+
+                return [
+                    ...defaultButtons,
+                    {
+                        id: "one_or_less_selected",
+                        label: "Ext one or less selected",
+                    }
+                ]
+            },
+        },
+    },
+});
+```
+The `defaultButtons` constant contains the buttons that will be rendered every time, no matter what condition is.
+On the line `const selections = await guestConnection?.host?.fragmentSelections?.getSelections?.() || [];` 
+we get information about current list of selected content fragments.
+
+Based on quantity of selected content fragments we rendered different set of buttons.
```

Comment:
Based on the quantity of selected content fragments, we render different sets of buttons.